app/scripts/views/components/vc/editableRowComponent.js

```diff
@@ -79,11 +79,11 @@
       },
       clickImport: function(_virtualStudy) {
         this.showmodal = false;
-        window.open(window.cbioURL + 'study?cohorts=' + _virtualStudy.virtualCohortID);
+        window.open(window.location.origin + window.location.pathname.substring(0, window.location.pathname.indexOf('/', 2)) + '/study?cohorts=' + _virtualStudy.virtualCohortID);
```

Comment:
Yes we could. But we need to set this variable wherever we initialized this component. Or we need to declare this variable in global top level js(may be include.js) in cbio so that we can use it for all components for iViz or session management component.